netpyne_ui/netpyne_geppetto.py

```diff
@@ -317,6 +317,15 @@ def getAvailablePlots(self):
         plots  = ["plotRaster", "plotSpikeHist", "plotSpikeStats","plotRatePSD", "plotTraces", "plotLFP", "plotShape", "plot2Dnet", "plotConn", "granger"]
         return [plot for plot in plots if plot not in simConfig.analysis.keys()]
 
+    def deleteParam(self, paramToDel):
+        logging.debug("checking if netParams."+paramToDel+" is not null")
+        if eval("netParams."+paramToDel) is not None:
```

Comment:
I would use something like `hasattr(netParams, 'cellParams')`. It is a good idea in general to avoid `eval` as much as possible